lib/material_design/railtie.rb

```diff
@@ -3,9 +3,10 @@
 module MyGem
   class Railtie < Rails::Railtie
     initializer "material_design.action_controller" do
-      ActiveSupport.on_load(:action_controller_base) do
+      ActiveSupport.on_load(:action_controller_base) do 
```

Comment:
Não é relacionado à sua PR mas to achando estranho o nome do `module` ser `MyGem`

Não deveria ser `MaterialDesign`?

Reply:
Alterei para `MaterialDesign` pois me parece uma boa prátiva. Como ainda não chamamos esse módulo em outros lugares, não vamos precisar alterar nada a mais no código.